fala/templates/adviser/other_region.html

```diff
@@ -5,15 +5,28 @@
 {% block content %}
   <div class="govuk-grid-column-full">
     <div id="google_translate_element" class="govuk-!-margin-bottom-6"></div>
-    {{ govukBackLink({
-      'text': "Back",
-      'href': url('adviser')
-    }) }}
+
+    {% if tailored_results %}
+      {{ govukBackLink({
+        'text': "Back",
+        'href': "/single-category-search?categories="+category_code,
+      }) }}
+    {% endif %}
+
     <h1 class="govuk-heading-xl">The postcode {{ postcode }} is in {{ region }}</h1>
     <p class="govuk-body">This search only covers England and Wales.</p>
     <p class="govuk-body">
       Find out about <a class="govuk-link" href="{{ link }}">Legal Aid in {{ region }}</a>
       or try a different search.
     </p>
+
+    {% if not tailored_results %}
+      <form action="/" method="get" novalidate>
```

Comment:
My question on this is just, do we need this to be in a form if we are clearing everything for the user?

Can the button be a Jinja2 button component?

Just a copy of another button I found in search.html that should help with the conversion. 
```
 {{ govukButton({
                'text': "Search",
                'type': "submit",
                'classes': "govuk-!-margin-bottom-2",
                'id': "searchButton",
                'name': "search"
              }) }}
```

Reply:
@psweeting1 👍 I have amended the button to use the Jinja2 button component. I have left it in a form for now as this was the existing behaviour and I didn't want to increase the scope of this ticket but maybe something we could consider in a future refactor?